torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1277,26 +1301,50 @@ void insertQuantDeQuantCall(
   } else {
     quantize_func = "quantize_per_tensor";
   }
-  Node* quant = g->create(at::Symbol::aten(quantize_func), inputs);
-  quant->output()->setDebugName(v->debugName() + ".quant");
-  g->insertNode(quant);
-
+  Node *quant, *choose_qparams;
+  
+  if (is_dynamic && !checkValueIsWeight(module, g, v)) {
+    std::string choose_qparams_func = "_choose_qparams_per_tensor";
+    auto reduce_range = g->insertConstant(false);
+    // choose_qparams_per_tensor has 2 outputs, (scale, zero_point).
+    choose_qparams = g->create(at::Symbol::aten(choose_qparams_func), {v, reduce_range}, 2);
```

Comment:
nit: please add a comment for 2 as well:
g->create(at::Symbol::aten(choose_qparams_func), {v, reduce_range}, /* num_outputs = */ 2);